mesh/src/main/java/no/nordicsemi/android/mesh/sensorutils/Energy32.java

```diff
@@ -0,0 +1,53 @@
+package no.nordicsemi.android.mesh.sensorutils;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.RestrictTo;
+
+/**
+ * The Energy 32 characteristic is used to represent a measure of energy in units of kilowatt-hours.
```

Comment:
Perhaps mention how it's encoded, not sure if that is done for other characteristics.

Reply:
The others doesn't mention encoding, and I think it's fine since you can check the specs. Mitigates human errors by not duplicating that.